.devcontainer/Dockerfile

```diff
@@ -1,10 +1,17 @@
 FROM ruby:3.2.1
 
 # install dependencies
-RUN apt-get update -qq && apt-get install -y imagemagick shared-mime-info libvips && apt-get clean
-
-RUN curl -sL https://deb.nodesource.com/setup_16.x | bash \
- && apt-get update && apt-get install -y nodejs && rm -rf /var/lib/apt/lists/* \
+RUN apt-get update -qq \
+ && apt-get install -y imagemagick shared-mime-info libvips \
+ && apt-get install -y ca-certificates curl gnupg \
+ && apt-get clean \
```

Comment:
As all the commands run together, they will be stored in a single layer in Docker Image.

The `apt-get clean` and `rm -rf /var/lib/apt/lists/*` can be performed at the end of the commands once rather than running them multiple times.